index.html

```diff
@@ -14,45 +14,255 @@
+          <p class="service">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="projectBTN"><button>See Project</button></div>
+        </article>
+
+        <article>
+          <img
+            class="card_image"
+            src="Snapshoot Portfolio2.png"
+            alt="project2 image"
+          />
+          <h3 class="projectName_holder">Multi-Post Stories</h3>
+          <div class="canopy">
+            <span>CANOPY</span><img src="Counter.png"alt="dot">
+            
+            <p>Back End Dev</p><img src="Counter.png"alt="dot">
+            
+            <p>2015</p>
+          </div>
+          <p class="service">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="projectBTN"><button>See Project</button></div>
+        </article>
+
+        <article>
+          <img
+            class="card_image"
+            src="Snapshoot Portfolio3.png"
+            alt="project3 image"
+          />
+          <h3 class="projectName_holder">Tonic</h3>
+          <div class="canopy">
+            <span>CANOPY</span><img src="Counter.png"alt="dot">
+            
+            <p>Back End Dev</p><img src="Counter.png"alt="dot">
+            
+            <p>2015</p>
+          </div>
+          <p class="service">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="projectBTN"><button>See Project</button></div>
+        </article>
+
+        <article>
+          <img
+            class="card_image"
+            src="Snapshoot Portfolio4.png"
+            alt="project4 image"
+          />
+          <h3 class="projectName_holder">Multi-Post Stories</h3>
+          <div class="canopy">
+            <span>CANOPY</span><img src="Counter.png" alt="dot">
+           
+            <p>Back End Dev</p><img src="Counter.png" alt="dot">
+            
+            <p>2015</p>
+          </div>
+          <p class="service">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="projectBTN"><button>See Project</button></div>
+        </article>
+      </section>
+
+      <section class="about">
+        <h2>About<br />Myself</h2>
```

Comment:
- [ ] Dear @Drew246, Just as mentioned by the previous reviewer, the `About Myself` text is on a single line, I can see you have added the `<be />` tag, which makes the `Myself` display on a new line, this is not as shown in the original design. Please fix this one for better alignment with the original design 

### Figma design 
![](https://user-images.githubusercontent.com/65298048/251858389-1439c2ab-1194-4c5b-b3f2-ea8b408fed4a.png)
### Your design 
![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/69077061/4d35034e-0900-4c28-9894-5988c15cb247)